OPCB/initPlayerLocal.sqf

```diff
@@ -79,9 +54,35 @@ jeff addaction ["Lights off", {
 [] spawn {
 	scriptName "ACE_advanced_fatigue_rebalance";		
 	while {true} do {
-			sleep 5;
-			ace_advanced_fatigue_anfatigue = 1;
+		sleep 5;
+		ace_advanced_fatigue_anfatigue = 1;
 	};
 };
 
-Hz_pers_clientReadyForLoad = true;
\ No newline at end of file
+Hz_pers_clientReadyForLoad = true;
+
+[] spawn {
+	waitUntil{
+		sleep 1;
+		_initDone = player getVariable ["InitDone", false];
+		_initDone
+	};
+
+	#include "data\whitelistedUnitTypes.sqf";
+
+	if (typeof player in _whitelistedUnitTypes) then {
+		_whiteListed = player getVariable ["WhiteListed", false];
```

Comment:
Are these variables going to persist after respawn?

Reply:
yes I tried.